src/views/homepage/components/learn-section/learn-section.module.css

```diff
@@ -0,0 +1,81 @@
+.learnSection {
+  --container-max-width: 1240px;
+  --padding-y: 64px;
+  --padding-x: 24px;
+
+  background-color: var(--token-color-vault-brand);
+}
+
+.intro {
+  padding: var(--padding-y) var(--padding-x);
+}
+
+.introInner {
+  --grid-template-columns: 1;
+
+  margin: 0 auto;
+  max-width: var(--container-max-width);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-template-columns), 1fr);
+  gap: 24px;
+
+  @media (--dev-dot-tablet-up) {
+    --grid-template-columns: 3;
+  }
+}
+
+.introMedia {
+  display: flex;
+  grid-column: 1 / -1;
+
+  @media (--dev-dot-tablet-up) {
+    grid-column: 3 / -1;
+    grid-row: 1;
+  }
+}
+
+.introContent {
+  grid-column: 1 / -1;
+
+  @media (--dev-dot-tablet-up) {
+    grid-column: 1 / 3;
+    max-width: 760px;
+  }
+}
+
+.introDescription {
+  margin: 12px 0 24px;
+
+  & a {
+    color: inherit;
+    text-decoration: underline;
+  }
```

Comment:
Not sure if you all have a class or component that manages this styling for content.

Reply:
Is this for the link that says "HashiCorp Cloud Engineer Certifications"? We have the `InlineLink` component that we use for content, but it has different styles than what you need here I think.

Reply:
Went ahead and made use of InlineLink. Will figure out color in polish task https://app.asana.com/0/1200641486648086/1202127264881989